editor/liveblocks.config.ts

```diff
@@ -134,42 +134,46 @@ export const {
       return []
     }
 
-    const room = liveblocksClient.getRoom(projectIdToRoomId(projectId))
-    if (room == null) {
-      return []
-    }
-
-    const storage = await room.getStorage()
-
-    const collabs = storage.root.get('collaborators') as LiveObject<{ [userId: string]: User }>
-    if (collabs == null) {
-      return []
-    }
-
-    const users = Object.values(collabs.toObject()).map((u) => u.toObject())
+    const users = await getAllUsersFromRoom(projectIdToRoomId(projectId))
```

Comment:
maybe I'm thinking wrongly about this one - but does this need to be updated when the project Id changes? (e.g. fork)

Reply:
getProjectID() always returns the current id, so this should survive a fork (and after a projectId change the room context is rerendered).

Reply:
btw shouldn't getProjectID use the editorState instead of reading off of window?

Reply:
I can not access editorState from the liveblocks resolveUsers function, only globals (and it is not a react component or hook, so I can not access the useEditorState hook neither)

Reply:
But in the long run we should implement a backend endpoint to retrieve users, and then this whole resolveUsers hack will become unnecessary.